webapp/src/components/intersections/ReportsPage.tsx

```diff
@@ -177,13 +217,14 @@ const Page = () => {
             ></Box>
           </Box>
           <ReportListTable
-            group={group}
+            group={true}
```

Comment:
question: Should there be a variable we're connecting this property to? If not, do we need to open it as an option?

Reply:
This was unused inside the table object - removed!